obs_img_utils/cli.py

```diff
@@ -177,17 +179,50 @@ def main(context):
     is_flag=True,
     help='Skip the image checksum validation.'
 )
+@click.option(
+    '--add-conditions-file',
+    type=click.STRING,
+    help='Specify conditions for the image from a file (with the conditions '
+         'in json format as a LIST)',
+    default=''
+)
+@click.option(
+    '--add-conditions-json',
```

Comment:
Is it worth a major version bump and an incompatible change for the extra clarity of the argument? `--add-conditions-json` vs. previously `--add-conditions`

Reply:
Fair point. I left the original options as they were and modified the new cli options to include an '-arg' suffix.